utilities/qneg_namelist.xml

```diff
@@ -77,11 +77,11 @@
                                                                         -->
 <!-- Qneg Parameterization -->
 
-  <entry id="print_qneg_warn">
```

Comment:
Should we change `print_qneg_warn` here?  I thought this is what a user would manually add to the `user_nl_cam` file, and clearly `pring_qneg_warn` is shorter than `control_for_negative_constituent_warning`, although it is also less descriptive.  Any thoughts?  Also pinging @cacraigucar to get her thoughts here too.

Reply:
I'm good either way. It's definitely shorter but, on the other hand, using the standard name might make things less complicated (one name for the variable).

Reply:
I would vote that we keep the names that users use as namelists not overly verbose (i.e. most CCPP standard names).  Also to me, `print_qneg_warn` is much more intuitive with what I know about the code than `control_for_negative_constituent_warning`